### PR TITLE
Add APIx1 CODEOWNER

### PR DESCRIPTION
**Pull request by Zuhairahmed**
This tag pulls current APIx1 on-call onto PR review

## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Global rule
 *	@themantissa
 
+# Maintained by the MongoDB APIx1 team
+* @mongodb/APIx1
+
```
Comment (themantissa) [inline .github/CODEOWNERS]: There are errors being reported by github on the entries below this line.   I'd just delete them as /examples should be the two above and the other is superseded by APIx1

**Review by themantissa — CHANGES_REQUESTED** [review]
See my comment regarding errors

File: .github/CODEOWNERS (hunk 1 above)
Comment (Zuhairahmed) [reply .github/CODEOWNERS]: makes sense, deleted.

**Review by martinstibbe — APPROVED** [review]
LGTM

**Review by themantissa — APPROVED** [review]
Looks good, ty!

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**